src/summary/cards/horizontal-section-card.component.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
```

Comment:
given that you are now using a `<table>` to display data, do we need this class?

Reply:
You are right. We don't need